app/components/lead_providers/partnerships/challenged_banner.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module LeadProviders
+  module Partnerships
+    class ChallengedBanner < BaseComponent
+      def initialize(partnership:)
+        @partnership = partnership
+      end
+
+      def render?
+        partnership.challenged?
+      end
+
+    private
+
+      attr_reader :partnership
+
+      def challenge_reason
+        t(partnership.challenge_reason, scope: "partnerships.challenge_reasons")
+      end
```

Comment:
`partnership.challenge_reason` is not a human readable text so it requires humanization. The text we display to the user on that page should match the text we display on the form, so we need certain reusability here (it is also used in the challenge email).

While I agree there is a cost, it is IMO fully justified by the gains - ability to test the component content without making them over-fragile by repeating hardcoded text. This significantly increases our confidence in our tests, reduces number of expensive feature tests, avoiding "we're not running our tests locally because it takes too long" or "the deployment process takes 30 minutes" situations.